lib/pages/error_page/error_page.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:fluffychat/pages/error_page/error_page_style.dart';
+import 'package:fluffychat/resource/image_paths.dart';
+import 'package:fluffychat/utils/platform_infos.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:go_router/go_router.dart';
+
+class ErrorPage extends StatelessWidget {
+  const ErrorPage({Key? key}) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      // Add invisible appbar to make status bar on Android tablets bright.
+      appBar: AppBar(
+        automaticallyImplyLeading: false,
+        elevation: 0,
+        backgroundColor: Theme.of(context).scaffoldBackgroundColor,
+      ),
+      extendBodyBehindAppBar: true,
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            PlatformInfos.isMobile
```

Comment:
```suggestion
     ResponsiveUtils.isMobile
```
- Also should create private class